### ISSUE TITLE
Delete Model with SoftDelete Behavior Breaks with Mapped Properties and Snapshots

### ISSUE THREAD

**Issue by virgofx** [phalcon/cphalcon#13302]
Deleting a model that is using the SoftDelete trait with snapshots results in an error:

`Column 'firstCamelCasedColumn' doesn't make part of the column map`

The problem is with the `columnMap` parameters passed into the setSnapshotData() and setOldSnapshotData().

https://github.com/phalcon/cphalcon/blob/master/phalcon/mvc/model/behavior/softdelete.zep#L102-L103
```
model->setSnapshotData(updateModel->getSnapshotData(), columnMap);
model->setOldSnapshotData(updateModel->getOldSnapshotData(), columnMap);
```

Removing the columnMap from the second parameter fixes the issue. That would be the expected behavior as there is no need to re-translate keys after getting old data and then resetting. Otherwise, there is an issue somewhere in the setting of snapshots with column maps.

/cc @Jurigag 

### Details

* Phalcon version: 3.3.1
* PHP Version: PHP 7.1.14
* Operating System: Ubuntu 16.04
* Installation type: Package Manager
* Server: Nginx

**Comment by virgofx** [comment]
@Jurigag Can you take a look and confirm the issue?

**Comment by virgofx** [comment]
Confirm fixed in 3.3.2

**Comment by sergeyklay** [comment]
I apologize @virgofx for completely overlooking this. Thank you for the report, and for helping us make Phalcon better.
